frontend/public/reducers/monitoring.ts

```diff
@@ -1,9 +1,5 @@
-import { connect } from 'react-redux';
-import { Map as ImmutableMap } from 'immutable';
 import * as _ from 'lodash-es';
 
-import { MonitoringAction, ActionType } from '../actions/monitoring';
-
 export const enum AlertStates {
```

Comment:
Here I wasnt sure if we want keep it in this reducer file or create a new one called eg. `alerts.ts`

Reply:
Actually, I think we should aim to move the monitoring Redux logic currently in `reducers/ui.ts` into a separate reducer, so that would mean into this file. So probably best to leave this file as is.

@spadgett @benjaminapetersen